### PR TITLE
latency breakdown

### PR DESCRIPTION
**Pull request by longbowlu**

### CODE REVIEW

File: crates/sui-core/src/transaction_orchestrator.rs [hunk 1]
```diff
@@ -397,73 +432,102 @@ where
...
+        let request_latency = HistogramVec::new_in_registry(
+            "tx_orchestrator_request_latency",
+            "Time spent in processing one Transaction Orchestrator request",
+            &["type"],
+            registry,
+        );
+        let wait_for_finality_latency = HistogramVec::new_in_registry(
+            "tx_orchestrator_wait_for_finality_latency",
+            "Time spent in waiting for one Transaction Orchestrator request gets finalized",
+            &["type"],
```
Comment (emmazzz) [inline crates/sui-core/src/transaction_orchestrator.rs:512]: Actually there's other code that might touch the system object, frenemies being one of them. Third party can also write Move functions that take in the system state object. Maybe we can add a label for the calling module of the transaction? This way we can see what's driving the high demand for system state obj: whether it's frenemies game, staking functions or someone else's module.

Comment (longbowlu) [reply crates/sui-core/src/transaction_orchestrator.rs:512]: Is staking/unstaking/switch gonna dominate txes that call 0x5? I don't think we should log every move call if we don't care about them. Another way is to log only staking related function calls, assuming there are only a few.

**Review by emmazzz — APPROVED** [review]
lgtm

**Comment by longbowlu** [comment]
tested locally, merging now

**Pull request merged**